src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="product">
+      <div class="product__card" data-qa="card">
+
+        <img class="picture" src="../src/images/imac.jpeg"
+        alt="image">
```

Comment:
it's not descriptive, for screan readers and search engins it's obvious that it is an image, it's more important to describe what is no this image- example 'APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)'